core/com.b2international.snowowl.api.rest.tests/pom.xml

```diff
@@ -23,22 +23,17 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.jetty.osgi.boot</id>
-								<versionRange>[8.1.0,8.2.0)</versionRange>
+								<versionRange>0.0.0</versionRange>
```

Comment:
Is this needed because of the difference between the "server" and "server + client" target platform versions of Jetty?

Reply:
No, they use the same Jetty version now. 
This is purely to remove the extra headache when you update Jetty versions.